codefellowship/codefellowship/src/main/java/com/codefellowship/controllers/CodefellowshipController.java

```diff
@@ -31,4 +31,15 @@ public String newPost(Model model) {
         return "newpost";
```

Comment:
Line 29: you need a getmapping for post before you have a postmapping to create posts.